tensilelite/Tensile/SolutionStructs/Solution.py

```diff
@@ -371,6 +371,14 @@ def assignProblemIndependentDerivedParameters(state, printRejectionReason: bool,
     state["tailLoopOptA"] = True
     state["tailLoopOptB"] = True
 
+    # Use nonDTL loads in DTL tail loop
+    state["NonDTLTailLoop"] = False
+    if (((state["ProblemType"]["DataType"].isHalf() or state["ProblemType"]["DataType"].isBFloat16()) and \
+        state["AssertSummationElementMultiple"] % 2 != 0) or \
```

Comment:
I suggest to check if TLU is false or not here.
Looks like DTL is not enabled in TLU=True case, but ASEM matters only if TLU=False.
It is better to add necessary check in case we enable DTL for TLU=True in the future.

Reply:
That makes sense, I've split `NonDTLTailLoop` for A/B separately and added the TLU checks.